src/components/badge/Badge.tsx

```diff
@@ -1,40 +1,59 @@
 import React from "react";
+import { EBadgePosition } from "./EBadgePosition";
 import { SBadge } from "./styles/SBadge";
-import { SBadgeIconContent } from "./styles/SBadgeIconContent";
 import { SBadgeIcon } from "./styles/SBadgeIcon";
-import { EBadgePosition } from "./EBadgePosition";
+import { SBadgeIconContent } from "./styles/SBadgeIconContent";
 
-export interface IBadge {
+export type IBadgeType = "regular" | "important" | "white" | "dark" | "color";
+export type IBadgeSizeType = "micro" | "small" | "medium";
+
+export interface IBadgeProps {
   value: string;
   position: EBadgePosition;
   disabled?: boolean;
+  type: IBadgeType;
+  size?: IBadgeSizeType;
 }
 
-export const Badge: React.FC<IBadge> = ({ ...props }) => {
+export const Badge: React.FC<IBadgeProps> = ({ children, ...props }) => {
```

Comment:
Prefer destructuring and named imports for react. Also badge is not working properly. Test here https://league-pxa6boiva-caisy.vercel.app/?path=/story/components-badge--position-left&args=type:color;value:11